src/vocabulary/model.rs

```diff
@@ -120,6 +143,11 @@ impl Vocabulary {
                     &Literal::<String>::new_lang_unchecked(creator, lang.clone()),
                 )?;
             } else if let Some(parent_id) = term.parent_id {
+                if terms_blocklist.contains(&parent_id) {
+                    terms_blocklist.insert(term.id);
+                    continue;
+                }
```

Comment:
It is guaranteed that the terms come in an order that the parents come first due to the ordering by id, right?

Reply:
Well, usually? ^^' But true, theoretically it is possible that `child.id < parent.id`...

Reply:
I'll think about that edge case.

Reply:
Fixed via https://github.com/serlo/serlo.org-database-layer/pull/141/commits/688c99cabb2ae677d85751670907247deff6f020